### DISCUSSION TITLE
start menu tiles color

### DISCUSSION THREAD

**Discussion by flowercatte** [valinet/ExplorerPatcher#2386] [Q&A]
Hello! When I'm using windows 10 tiles on the start menu, they're the monochrome color that was pushed in a later update. is there a way to configure them to show the accent color as the background like in the older style of windows 10?

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio

**Reply by XuanChen1234** [reply]
Page from: https://github.com/valinet/ExplorerPatcher/discussions/2487
![image](https://github.com/valinet/ExplorerPatcher/assets/140480110/acb0b6c3-bbf4-4d67-b776-e00eadc7459d)
